testsys/src/run_aws_ecs.rs

```diff
@@ -101,6 +105,35 @@ pub(crate) struct RunAwsEcs {
     /// Name of the pull secret for the EC2 provider image.
     #[structopt(long)]
     ec2_provider_pull_secret: Option<String>,
+
+    /// Perform an upgrade downgrade test.
+    #[structopt(long, requires_all(&["starting-version", "upgrade-version"]))]
+    upgrade_downgrade: bool,
+
+    /// Starting version for an upgrade/downgrade test.
+    #[structopt(long)]
+    starting_version: Option<String>,
```

Comment:
Can all of these require `--upgrade-downgrade`? i.e. can we avoid accepting something like `--starting-version` when upgrade_downgrade is false?